### ISSUE TITLE
[Lecture10][0616] About nesterov 

### ISSUE THREAD

**Issue by Kwonminjeong0** [PiLab-CAU/ComputerVision-2401#49]
![결합본](https://github.com/PiLab-CAU/ComputerVision-2401/assets/92988028/f751fabc-15d6-428e-9801-dada7aa77026)

I know that "Nesterov Momentum anticipates the next step's location by first applying momentum in the current direction from the current position, then computing the gradient at that position, and moving in that direction once more."

I think I have some understanding of what "nesterov" is. However, I don't understand what is the status of "if nesterov" in the pseudo code in the image above. 
My understanding is that if nesterov is applied, it should be applied in every step, and if not, it should not be applied at all. I don't know how the if nesterov state can exist..

I would be grateful if you could tell me about "if nesterov"...

**Comment by yjyoo3312** [comment]
@Kwonminjeong0 Thank you for the comment!

I am afraid my understanding of the question is correct. In PyTorch SGD ([link](https://pytorch.org/docs/stable/generated/torch.optim.SGD.html)), we can select an ordinary momentum method or nesterov momentum method. 

When we initialize optimizer class, we can set the Nesterov option to True or false, and If true, nesterov momentum will be applied everytime.
